configure tests for availability of alloca.h, but
it's used unconditionally here.

--- libtest/lite.h.orig	Sun Nov  8 20:10:49 2015
+++ libtest/lite.h	Sun Nov  8 20:10:32 2015
@@ -53,7 +53,7 @@
 
 #if defined(WIN32)
 # include <malloc.h>
-#else
+#elif defined(HAVE_ALLOCA_H)
 # include <alloca.h>
 #endif
 
